components/forms/Login/index.jsx

```diff
@@ -20,13 +23,7 @@ const Login = () => {
         >
           Email
         </label>
-        <input
-          className="w-full h-14 bg-white/20 text-white text-base-large rounded-xl px-3 py-1.5 focus:outline-none focus:shadow-none"
-          id="email"
-          data-test-id={loginFormTestIds.emailField}
-          type="email"
-          required
-        />
+        <Input id="email" type="email" dataTestId={loginFormTestIds.emailField} required={true} />
```

Comment:
How about just `required`?

```suggestion
        <Input id="email" type="email" dataTestId={loginFormTestIds.emailField} required />
```

Reply:
done in d129258d877c1e23ba31313cc0390ad9627eb192